kajas/src/app/edit-artwork/edit-artwork.component.ts

```diff
@@ -0,0 +1,166 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { HttpClient } from '@angular/common/http';
+import { Router, ActivatedRoute } from '@angular/router';
+import { SessionStorageService } from 'angular-web-storage';
+import { ArtworkService } from '../../services/artwork.service';
+
+@Component({
+  selector: 'app-edit-artwork',
+  templateUrl: './edit-artwork.component.html',
+  styleUrls: ['./edit-artwork.component.css'],
+})
+export class EditArtworkComponent implements OnInit {
+  artworkForm: FormGroup;
+  imageUrl: string | null = null;
+  showModal = false;
+  modalMessage = '';
+  artwork: any;
+  titlePlaceholder = '';
+  dateCreatedPlaceholder = '';
+  descriptionPlaceholder = '';
+
+  constructor(
+    private fb: FormBuilder,
+    private http: HttpClient,
+    private router: Router,
+    private route: ActivatedRoute,
+    private sessionStorage: SessionStorageService,
+    private artworkService: ArtworkService
+  ) {
+    this.artworkForm = this.fb.group({
+      title: ['', Validators.required],
```

Comment:
### Always use the object form in creating validators.
[https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g270ee9174f4_0_43](url)